src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
```

Comment:
Since you're naming the other classes as 'Events', 'Deadlines', and 'Todos', might be nice to change this to 'Tasks' as well.